Resources/Prototypes/_RMC14/Entities/Structures/Furniture/linenbin.yml

```diff
@@ -0,0 +1,87 @@
+- type: entity
+  parent: BaseStructureDynamic
+  id: CMLinenBin
+  suffix: Empty
+  name: linen bin
+  description: A linen bin. It looks rather cosy.
+  components:
+  - type: Anchorable
+  - type: Transform
+    anchored: true
+  - type: InteractionOutline
+  - type: Damageable
+    damageContainer: Inorganic
+    damageModifierSet: Metallic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 150
+      behaviors:
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+    - trigger:
+        !type:DamageTrigger
+        damage: 100
+      behaviors:
+      - !type:EmptyAllContainersBehaviour
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          CMSheetMetal1:
+            min: 1
+            max: 1
+  - type: Sprite
+    noRot: true
+    sprite: _RMC14/Structures/Furniture/linen_bin.rsi
+    layers:
+    - state: linenbin_empty
+      map: [ "emptyLayer" ]
+    - state: linenbin_half
+      map: [ "partialLayer" ]
+    - state: linenbin_full
+      map: [ "fullLayer" ]
+  - type: ItemSlots
+  - type: CMItemSlots
+    count: 10
```

Comment:
20 is fine if that's how it works in 13, there's not really anything bad that it would enable by having that many